config.go

```diff
@@ -618,10 +618,23 @@ type Config struct {
 	StateCallback        HTTPCallback
 	StateReportFrequency int
 
+
 	// Report error via an HTTP callback. The Payload field will contain the ErrorType,
 	// ErrorMessage and the StateDump.
 	ErrorCallback HTTPCallback
 
+	// Report when ghostferry is entering cutover
+	CutoverLock   			HTTPCallback
+
+	// Report when ghostferry is finished cutover
+	CutoverUnlock 			HTTPCallback
+
+	// These two values configure the amount of times Ferry should attempt to
+	// retry acquiring the cutover lock, and for how long the Ferry should wait
```

Comment:
Probably should add before:

If the server for CutoverLock returns a non OK status, then these two values ... <carry on>